pom.xml

```diff
@@ -15,8 +14,13 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <surefire.rerunFailingTestsCount>0</surefire.rerunFailingTestsCount>
+        <commons.version>1.10</commons.version>
+        <httpcomponents.version>4.5.1</httpcomponents.version>
+        <script-security.version>1.17</script-security.version>
```

Comment:
Typically we would not bother introducing a property for a version which is used in only one place.